hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -144,6 +168,33 @@ public HoodieTestTable addCommit(String instantTime) throws Exception {
     return this;
   }
 
+  public HoodieCommitMetadata createCommitMetadata(WriteOperationType operationType, String commitTime,
+                                                   Map<String, List<Pair<String, Integer>>> partitionToFileIdMap) {
```

Comment:
should try encapsulate data structure like `partitionToFileIdMap` within `HoodieTestState` and  make it invisible to users. It's not easy to grasp and keep recalling what info is kept in the Map. And more friction of using it in an API